### ISSUE TITLE
Make a More Delicate Navigation on Resource Map

### ISSUE THREAD

**Issue by chelseaerdner** [saxifrage/caac-map#65]
I am not sure if there is an existing issue for this but I could not find it. If so, please feel free to close this one and point me to it's location @timothyfcook. But, we have mentioned before that the map's navigation looked too bulky and took too much space on the screen.

I reduced it's size and made it more consistent as a whole. I am open to other thoughts/options if anyone has any ideas. ???

![navigation_09212015](https://cloud.githubusercontent.com/assets/11842603/9997611/2689ab8e-605b-11e5-8b46-1e8abc7b46b1.png)

**Comment by timothyfcook** [comment]
Relates to #33 and #36 but is fine as a separate issue. Likely once the menu work is finished by @cameronscott137 with help from @whit537 these will all be resolved.

Cameron can implement this new design.

**Comment by chelseaerdner** [comment]
Okay, take two. I am open to suggestion here but I put together a new approach to the navigation to keep it off canvas. We have run into a lot of issues with the cards hidden under an exposed nav so I added a hamburger icon (similarly styled to the zoom function). If a user clicks that icon, a side bar nav will expose the main topics. I think this approach compliments the new way we are handling the card layouts. 

Like I said, I am open to thought and opinions. 

Thanks!

![screen shot 2015-10-27 at 10 34 24 am](https://cloud.githubusercontent.com/assets/11842603/10761110/5411631a-7c96-11e5-935b-400ac612a35c.png)

![screen shot 2015-10-27 at 10 05 40 am](https://cloud.githubusercontent.com/assets/11842603/10761117/5a7d70c2-7c96-11e5-95b5-f657ed41722f.png)

**Comment by timothyfcook** [comment]
This could work. Although we're really boxing in the map and losing space fast....

Maybe we could keep the menu on the top of bottom running horizontally, but as soon as you make a choice from the menu the menus shrink and fold up so they don't cover the map.

**Comment by chadwhitacre** [comment]
Closing in favor of https://github.com/saxifrage/cityasacampus/pull/328.
